internal/service/domain/auth/user.go

```diff
@@ -66,3 +66,12 @@ func (s *AuthService) LogoutRequest(ctx context.Context, logoutRequest *domain.L
 
 	return nil
 }
+
+func (s *AuthService) GetAllUsers(ctx context.Context) ([]*domain.User, error) {
```

Comment:
In the other services we call this method just »GetAll«. 
To make it uniform everywhere, I would suggest that we name it GetAll here as well? Would that be okay?